addons/captives/functions/fnc_canApplyHandcuffs.sqf

```diff
@@ -20,7 +20,7 @@ params ["_unit", "_target"];
 //Check sides, Player has cableTie, target is alive and not already handcuffed
 
 (GVAR(allowHandcuffOwnSide) || {(side _unit) != (side _target)}) &&
-{"ACE_CableTie" in (_unit call EFUNC(common,uniqueItems))} &&
+{((_unit call EFUNC(common,uniqueItems)) select {_x isKindOf "ACE_CableTie"}) isNotEqualTo []} &&
```

Comment:
Not gonna work, need to specify CfgWeapons for the lookup.

Reply:
Second time I've done that, my bad. Will fix